fuel-tx/src/transaction/types/script.rs

```diff
@@ -45,16 +50,51 @@ pub struct ScriptMetadata {
     pub script_data_offset: usize,
 }
 
+#[derive(Clone, Default, Derivative)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "serde", serde(transparent))]
+#[derive(fuel_types::canonical::Deserialize, fuel_types::canonical::Serialize)]
+#[derivative(Eq, PartialEq, Hash, Debug)]
+pub struct ScriptCode {
+    #[derivative(Debug(format_with = "fmt_truncated_hex::<16>"))]
+    pub bytes: Vec<u8>,
+}
+impl From<Vec<u8>> for ScriptCode {
+    fn from(bytes: Vec<u8>) -> Self {
+        Self { bytes }
+    }
+}
+impl Deref for ScriptCode {
+    type Target = Vec<u8>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.bytes
+    }
+}
+impl DerefMut for ScriptCode {
+    fn deref_mut(&mut self) -> &mut Self::Target {
+        &mut self.bytes
+    }
+}
+
+#[cfg(feature = "da-compression")]
+impl fuel_compression::Compressible for ScriptCode {
+    type Compressed = fuel_compression::RegistryKey;
+}
```

Comment:
We also need to implement `CompressibleBy` manually by using `CompressionContext`

Reply:
Apparently not needed. Removed `De/CompressionContext` traits  in 9b33143.